src/pages/scheduler/components/share/ShareLinkOptions.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from 'react';
+
+import { CopyIcon } from '@chakra-ui/icons';
+import { Button, Heading, HStack, Icon, Input, useClipboard, useToast } from '@chakra-ui/react';
+import { HiLink } from 'react-icons/hi';
+import {
+  EmailShareButton,
+  EmailIcon,
+  FacebookShareButton,
+  FacebookIcon,
+  TelegramShareButton,
+  TelegramIcon,
+  WhatsappShareButton,
+  WhatsappIcon,
+  TwitterShareButton,
+  TwitterIcon,
+} from 'react-share';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+
+type SocialMediaButtons_Props = {
+  slug: string;
+};
+
+const SocialMediaButtons = ({ slug }: SocialMediaButtons_Props) => {
+  return (
+    <HStack justify="center">
+      <EmailShareButton children={<EmailIcon size={50} round={true} />} url={slug} />
+      <FacebookShareButton children={<FacebookIcon size={50} round={true} />} url={slug} />
+      <TelegramShareButton children={<TelegramIcon size={50} round={true} />} url={slug} />
+      <WhatsappShareButton children={<WhatsappIcon size={50} round={true} />} url={slug} />
+      <TwitterShareButton children={<TwitterIcon size={50} round={true} />} url={slug} />
+    </HStack>
+  );
+};
+
+type CopyLinkUrl = {
```

Comment:
```suggestion
type CopyLinkUrlProps = {
```
`CopyLinkUrl` is already defined below